<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseView</class>
 <widget class="QWidget" name="DatabaseView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>447</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" colspan="2">
    <widget class="QComboBox" name="dbStructCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="statusTip">
      <string comment="Statusbar text when hovering the DB description combobox list">Select the database structure to use and then click &quot;Load&quot;</string>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="dbLoadClose">
     <property name="statusTip">
      <string comment="Statusbar text when hovering the button next to DB description DLL combobox">Load/Unload the selected database structure</string>
     </property>
     <property name="text">
      <string notr="true">Loaded</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="searchButton">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="statusTip">
      <string comment="Statusbar text when hovering the search button">Search the written text from the current row</string>
     </property>
     <property name="text">
      <string comment="Button text">Search</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="dbStructLabel">
     <property name="text">
      <string comment="Label next to DB description combobox">Choose database structure:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="progressLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string comment="Label next to progressbar">Progress:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="searchColumnCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="statusTip">
      <string comment="Statusbar text when hovering the search column combobox">Select the column to be searched</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QTableView" name="databaseTable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="searchTextEdit">
     <property name="statusTip">
      <string comment="Statusbar text when hovering the search editfield">Write the text to be searched</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="encodingCombo">
     <property name="statusTip">
      <string comment="Statusbar text when hovering the encoding combobox">The locale to use to display text data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="encodingLabel">
     <property name="text">
      <string comment="Label next to encoding combobox">Encoding:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
